FabricExample/package.json

```diff
@@ -35,6 +35,7 @@
     "@types/react-test-renderer": "^18.0.0",
     "babel-jest": "^29.2.1",
     "eslint": "^8.19.0",
+    "eslint-plugin-ft-flow": "^2.0.3",
```

Comment:
Why we need this?

Reply:
I was sloppy and committed this change. In current commit it is removed.